lib/food.js

```diff
@@ -0,0 +1,38 @@
+const API = 'https://qs-be.herokuapp.com/api/v1/'
+// const LOCAL ='http://localhost:3000/api/v1/foods'
+
+$ = require('jquery')
+
+const Food = class Food {
+
+  getAll() {
+    return $.ajax({
+    method: 'GET',
+    url: API + 'foods',
+    })
+    .done(makeFoodTable)
+    .fail( (error) => {
+      console.error(error)
+    });
+  };
```

Comment:
I'm having trouble coming up with a way to test this, mostly because this sequence of events doesn't ever really return anything so I'm not quite sure what I could assert. Do you have any suggestions or ideas about the way I'm structuring this that might make unit testing possible? @case-eee @neight-allen